ios/sdk/base/HippyBridge.mm

```diff
@@ -84,7 +84,12 @@ void HippyRegisterModule(Class moduleClass)
         name = NSStringFromClass(cls);
     }
     if ([name hasPrefix:@"Hippy"] || [name hasPrefix:@"hippy"]) {
-        name = [name substringFromIndex:5];
+        //an exception,QB uses it
+        if ([name isEqualToString:@"HippyIFrame"]) {
+        }
+        else {
+            name = [name substringFromIndex:5];
```

Comment:
[boundary condition] the length of variable 'name' should be considered when it's less than 5.